prac_05/wimbledon.py

```diff
@@ -0,0 +1,42 @@
+from itertools import count
+
+FILE_PATH = "wimbledon.csv"
+CHAMPION_INDEX = 2
+CHAMPION_COUNTRY_INDEX = 3
+def main():
+    competitions = []
+    read_competitions(competitions)
+    champion_to_wins = get_champions_to_wins(competitions)
+    countries = get_winning_countries(competitions)
+    display_stats(champion_to_wins, countries)
+
+def display_stats(champions_to_wins, countries):
+    print("Wimbledon Champions:")
+    for champion,wins in champions_to_wins.items():
+        print(f"{champion} {wins}")
+    print(f"These {len(countries)} countries have won Wimbledon:")
+    print(", ".join(countries))
```

Comment:
Are you sure the output of this print statement matches the sample output? (Sorting)